pallets/payment/Cargo.toml

```diff
@@ -38,6 +38,7 @@ std = [
 	'sp-std/std',
 	'scale-info/std',
 	'orml-traits/std',
+	'orml-tokens/std',
```

Comment:
The CI check failed, this is likely the culprit since `orml-tokens` is only a dev dependency.
```suggestion
```